changelog/2603.feature.rst

```diff
@@ -1 +1,3 @@
-MapCube is now deprecated. Please use MapSequence.
+MapCube has been renamed to MapSequence.
+MapCube does not actually work on cubes but a sequence of maps.
```

Comment:
This is  MapCube. Also double backticks around class names and full links to MapSequence.